lametro/models/cms.py

```diff
@@ -57,3 +59,75 @@ def content(self):
             + f"<span class='button button-small alert-{self.type}'>{self.get_type_display()}</span>"
             + "</div>"
         )
+
+
+class CheckboxSelectMultipleList(forms.CheckboxSelectMultiple):
+    def format_value(self, value):
+        if isinstance(value, str):
+            return value.split(",")
+
+
+class EventNotice(models.Model):
+    """
+    A message that will show up on any event detail pages that match
+    any of the event conditionals applied to each message.
+    """
+
+    include_in_dump = True
+
+    def __str__(self):
+        return f"Notice on {', '.join(self.broadcast_conditions)} and {', '.join(self.comment_conditions)} Events"
+
+    BROADCAST_CONDITION_CHOICES = [
+        ("future", "Future events"),
+        ("upcoming", "Upcoming events"),
+        ("ongoing", "Ongoing events"),
+        ("concluded", "Concluded events"),
+    ]
+    COMMENT_CONDITION_CHOICES = [
+        ("accepts_live_comment", "Events that accept live public comments"),
+        ("accepts_comment", "Events that accept public comments when not live"),
+        ("accepts_no_comment", "Events that do not accept public comments at all"),
+    ]
```

Comment:
Hm, I'm a little confused by the comment conditions. I think all events accept comment?

Reply:
Ah okay, I was basing this off of the below section in the event template. Is this is a holdover from a time when not all events accepted comment, or maybe I'm just misinterpreting the below conditions?

https://github.com/Metro-Records/la-metro-councilmatic/blob/431a943513bca38620de620cd9c19ad5067c3280/lametro/templates/event/event.html#L109-L116

Reply:
Looks like some committee meetings don't accept comments: https://github.com/Metro-Records/la-metro-councilmatic/pull/823/files

But that's determined on the back end – this logic shouldn't be exposed in the CMS!

Reply:
Gotcha, I've removed it from the cms!

In that case, what do you think about me bringing back the public comment instructions in the template then? Since it says different things depending on which template is being rendered, and the same "upcoming" notice would render across all those templates regardless of whether the event accepted comment or not.

Reply:
Hmmm.................................. maybe you were right to include the comment conditions after all. Can we change the labels to:

- Events that accept public comment before and during the meeting
- Events that accept public comment before but not during the meeting
- Events that do not accept public comment

Reply:
Sorry for the run around!

Reply:
No worries! It's an easy adjustment and I could've explained myself more haha.